src/vm/util.hpp

```diff
@@ -810,9 +810,18 @@ inline void UnsafeTlsFreeForHolder(DWORD* addr)
 typedef Holder<DWORD*, DoNothing<DWORD*>, UnsafeTlsFreeForHolder> TlsHolder;
 
 // A holder for HMODULE.
-FORCEINLINE void VoidFreeLibrary(HMODULE h) { WRAPPER_NO_CONTRACT; CLRFreeLibrary(h); }
+FORCEINLINE void VoidFreeNativeLibrary(NATIVE_LIBRARY_HANDLE h)
```

Comment:
Should we test `h` for NULL? `dlclose` segmentation faults when input is null: [`dlopen`/`dlerror`/`dlclose` experiment](https://wandbox.org/permlink/Ghm7YjwU4pBAlTT2).

Reply:
Good catch. Thanks! https://github.com/dotnet/coreclr/pull/24685